src/backup.c

```diff
@@ -1343,6 +1362,9 @@ open_dir_file(backup_job_context_t *bjc)
 		if (update_file_pos(bjc->fd, &bjc->byte_count_file, &bjc->byte_count_job,
 					&bjc->status->byte_count_total) < 0) {
 			pthread_mutex_unlock(&bjc->status->dir_file_init_mutex);
+			cf_free(bjc->fd);
+			bjc->fd = NULL;
+			err("New directory file %s, has less than 0 bytes written", file_path);
```

Comment:
What is less than 0 bytes? A nibble? A bit?

Reply:
yeah that was confusing. changed

Reply:
I would have changed to ordering to "Failed to get file position from new directory file %s"